src/rotation.zig

```diff
@@ -112,6 +122,9 @@ pub const RotationMode = struct { // MARK: RotationMode
 
 	model: *const fn(block: Block) u16 = &DefaultFunctions.model,
 
+	// Rotates block data 90 degrees counterclockwise round Z axis.
```

Comment:
The doc comment is no longer correct.
```suggestion
	// Rotates block data counterclockwise around the Z axis.
```